.buildkite/code.pipeline.yml

```diff
@@ -203,7 +203,7 @@ steps:
   ###############
   - label: E2E tests
     parallelism: 7
-    timeout_in_minutes: 9
+    timeout_in_minutes: 12
```

Comment:
Can probably change this back before merging.

Reply:
Yes, this is still a WIP :)